.github/workflows/ci-release.yaml

```
@@ -0,0 +1,112 @@
+name: Release CI
+
+on:
+  workflow_dispatch:
+    inputs:
+      version-to-bump:
+        type: choice
+        required: true
+        description: "Select which part of the version to bump and release"
+        options:
+          - patch
+          - minor
+          - major
+          - prepatch
+          - preminor
+          - premajor
+          - prerelease
+
+jobs:
+  prepare-release:
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.variables.outputs.version }}
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3.0.0
+
+      - name: Set up NodeJs
+        uses: actions/setup-node@v3.1.1
+        with:
+          node-version: '16'
+          cache: 'npm'
+
+      - name: Setup Environment
+        id: variables
+        run: |-
+          git config user.name "dependency-track-bot"
+          git config user.email "dependency-track-bot@owasp.com"
```

Just curious: Why is this necessary? How does this behave compared to [what's done in the Dependency-Track pipeline](https://github.com/DependencyTrack/dependency-track/pull/1419/files#diff-eecbed2b66fe9be749f388e527e472f10b086c8cced6e90c234c8a23be5b8798R88-R93) via the `gh` CLI? Do we have to create that user first?

---

well, this was necessary to quiet `npm version`, which implicitly commits the version bump and would complain about an unset committer otherwise.

However, this can be worked around by telling npm not to implicitly commit, and committing via `gh` similar to the backend CI.

---

Ah alright. Maybe we should change the username and email to something that more directly indicates that a GH action did the commit: https://github.com/actions/checkout/issues/13#issuecomment-724415212

It looks like 
```suggestion
          git config user.name "github-actions[bot]"
          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
```

does the trick.

---

yeah, the values here where more a placeholder than anything real.